frontend/src/components/episode/components/timeline-item/timeline-item.tsx

```diff
@@ -40,7 +51,7 @@ const TimelineItem: React.FC<Props> = ({
         onClick={handleGoToItemTimestamp}
         className={styles.timelineItemBtn}
       />
-      <div className={styles.timelineItemContent}>
+      <div style={{ left: `${offsetBlockComment}px` }} className={styles.timelineItemContent}>
```

Comment:
```suggestion
      <div style={commentOffset < 50 ? { left: 0 } : { right: 0 }} className={styles.timelineItemContent}>
```
This change with suggested changes in style will be more simple (or some variants of them)

Reply:
We don't need it. Just add styles that we want